### PR TITLE
Upgrade to SBT 1

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -252,10 +241,7 @@ lazy val docs = project
       "extref.akka-http-docs.base_url" -> s"http://doc.akka.io/docs/akka-http/${Dependencies.AkkaHttpVersion}/%s.html",
       "extref.java-api.base_url" -> "https://docs.oracle.com/javase/8/docs/api/index.html?%s.html",
       "scaladoc.akka.base_url" -> s"http://doc.akka.io/api/akka/${Dependencies.AkkaVersion}",
-      "scaladoc.akka.management.http.base_url" -> {
-        if (isSnapshot.value) Paths.get((target in paradox in Compile).value.getPath).relativize(Paths.get(unidocTask.value.head.getPath)).toString
-        else s"http://developer.lightbend.com/docs/api/akka-management/${version.value}"
-      },
+      "scaladoc.akka.management.http.base_url" -> s"http://developer.lightbend.com/docs/api/akka-management/${if (isSnapshot.value) "snapshot" else version.value}",
```
Comment (raboof) [inline build.sbt]:
```suggestion
      "scaladoc.akka.management.http.base_url" -> s"https://developer.lightbend.com/docs/api/akka-management/${if (isSnapshot.value) "snapshot" else version.value}",
```

File: project/Dependencies.scala [hunk 2]
```diff
@@ -143,7 +143,15 @@ object Dependencies {
       "com.typesafe.akka" %% "akka-distributed-data"              % AkkaVersion     % "test"
     )
   )
-
+  
+ val ClusterGrpc = Seq(
```
Comment (raboof) [inline project/Dependencies.scala]:
```suggestion
  val ClusterGrpc = Seq(
```

**Comment by patriknw** [comment]
should we defer this until after the 1.0.0 release to not risk anything, since we are on final RC and about to release any day now

**Comment by chbatey** [comment]
Yep, let's hold off on this

**Comment by raboof** [comment]
(retriggering Travis because #514 started failing there recently, and I'm curious whether it also now fails for this branch or perhaps this branch even fixes the issue)

**Comment by chbatey** [comment]
Shall we merge this now @raboof ?

**Pull request merged**